packages/react-components/priority-overflow/src/overflowManager.ts

```diff
@@ -103,7 +137,7 @@ export function createOverflowManager(): OverflowManager {
       0,
     );
 
-    const overflowMenuSize = invisibleItemQueue.size() > 0 && overflowMenu ? getOffsetSize(overflowMenu) : 0;
+    const overflowMenuSize = isOverflowMenuVisible() && overflowMenu ? getOffsetSize(overflowMenu) : 0;
```

Comment:
same here

```suggestion
    const overflowMenuSize = isOverflowMenuVisible() ? getOffsetSize(overflowMenu) : 0;
```

Reply:
unfortunately not for typescript :( - I could make the `isOverflowMenuVisible` helper a type assertion too, but it would be kinda strange considering that the variable is already in scope

Reply:
Okay then I would rename the function and leave out the check for the overflow menu, eg hasOverflowItems